CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+##  v1.10.0 (12/14/2020)
+
+**Common**
+
+* Added a shared_mutex class.
+* Added status logging of the network requests on iOS platform.
+* Fixed the proxy warming message on Android platform.
+* Added `Open` and `Close` API to `olp::cache::DefaultCache`. It can be used to open or close individual caches.
+* `olp::client::ApiLookupClient` now caches the `olp::client::OlpClient` instances internaly.
+* Added merge of the same requests in `olp::client::OlpClient` class. Internally it merges te same requests by URL and with empty payload.
+* Removed the unnecessary sleep in `olp::http::NetworkCurl` which forced all requests to be at least 100 ms.
+* Added downloaded size and time of execution for requests in `olp::http::NetworkCurl`.
+
+**olp-cpp-sdk-authentication**
+
+* Added a new GetMyAccount API. It can retrieve user informarmation for the issued access_token.
+* Expose the Crypto class and methods used by module internaly.
+
+**olp-cpp-sdk-dataservice-read**
+
+* Fixed the data race inside TaskSink class.
+* Adapt StreamLayerClient, CatalogClient to use TaskSink.
+* Fixed the expiration of the latest version. The user provided expiration was not propagated to cache.
```

Comment:
Please add the scenarios, i.e. We always go online first then save to cache so that CacheOnly users always find the latest catalog version valid in cache. If a new version is available online we update the value in cache to the new catalog version.

Reply:
It was already specified in the prev. release